src/components/network/network-table.js

```diff
@@ -253,21 +281,39 @@ const NetworkTable = (props) => {
                         width: 400,
                         label: intl.formatMessage({ id: 'ID' }),
                         dataKey: 'id',
+                        headerStyle: {
```

Comment:
We should refactor this to not have to copypaste it everywhere

Reply:
Done